mscore/inspector/inspectorBase.cpp

```diff
@@ -544,7 +544,11 @@ void InspectorBase::mapSignals(const std::vector<InspectorItem>& il, const std::
                         resetButton->setIcon(*icons[int(Icons::reset_ICON)]);
                         connect(resetButton, &QToolButton::clicked, [=] { resetClicked(i); });
                         Sid sidx = inspector->element()->getPropertyStyle(ii.t);
-                        if (sidx != Sid::NOSTYLE) {
+                        // S button for fingering placement is bugged and proposed to be hidden
+                        // it can be brought back once the values of fingering placement are changed to
+                        // "Auto, Above, Below", "Auto" as default
```

Comment:
I wouldn't assume this exact solution is going to be implemented, it's one possible idea though.

Reply:
But whatever the solution is, it requires an overhaul of the design, right?